### ISSUE TITLE
picker title format request...

### ISSUE THREAD

**Issue by Neptunium1129** [probablykasper/date-picker-svelte#81]
![image](https://github.com/probablykasper/date-picker-svelte/assets/32185134/ca1165ed-4887-406f-b580-0833fa3fadcb)

need an option to change the location format title

now : mm - yyyy 
kor : yyyy - mm (2023 - 11)

In Korea, when reading the calendar, they often start with the year. For example, instead of saying "April 5th, 2023," they might say "2023 April 5th." This format prioritizes the year before the month and day.

**Comment by Neptunium1129** [comment]
example : 
![image](https://github.com/probablykasper/date-picker-svelte/assets/32185134/37a816fc-73f3-489d-8f31-6105fe9bbfac)
lang : ko
https://raw.githack.com/mymth/vanillajs-datepicker/v1.3.4/demo/index.html

**Comment by probablykasper** [comment]
Makes sense. From that datepicker, `zh-TW`, `zh-CN`, `ja`, `ko` and `ar` have the year first. Would be good to find a full list somewhere.

Implementation: Add a `monthFirst` property to `InnerLocale`. Add `code` property to `DateFnsLocale` object and use that to infer `monthFirst` in `localeFromDateFnsLocale()`

**Comment by Neptunium1129** [comment]
Can I see an example?..

**Comment by Neptunium1129** [comment]
> Makes sense. From that datepicker, `zh-TW`, `zh-CN`, `ja`, `ko` and `ar` have the year first. Would be good to find a full list somewhere.
> 
> Implementation: Add a `monthFirst` property to `InnerLocale`. Add `code` property to `DateFnsLocale` object and use that to infer `monthFirst` in `localeFromDateFnsLocale()`

Can I see an example?

**Comment by sinbino** [comment]
I need this too.
